### PR TITLE
teams/docs: a vote for Icemap as the committer

### PR DESCRIPTION
**Pull request by qiancai**
As stated in the title.

### CODE REVIEW

**Review by Oreoxmt — APPROVED** [review]

**Review by lilin90 — APPROVED** [review]
/lgtm

File: votes/0780-Icemap-as-docs-committer.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+The vote will be open for at least 6 days unless there is an objection or not enough votes.
+
+## Scope
+
+* Team: Docs
+
+## Result
+
+To-Be-Added
```
Comment (qiancai) [inline votes/0780-Icemap-as-docs-committer.md:31]:
```suggestion
Approved by 2 binding votes and 1 non-binding votes:

* lilin90 (binding)
* qiancai (binding)
* Oreoxmt (non-binding)
```

**Review by zhangyangyu — APPROVED** [review]

**Pull request merged**